source/networking/neutron-with-existing-external-network.html.md

```diff
@@ -7,11 +7,15 @@ authors: dneary, fale, javexed, mangelajo, mrunge, rbowen
 
 Many people have asked how to use packstack --allinone with an existing external network. This method should allow any machine on the network to be able to access launched instances via their floating IPs. Also, at the end of this message, there are some ideas for making this process better that I thought we could discuss.
 
-These instructions have been tested on Centos 7.
+These instructions have been tested on CentOS 7 and CentOS 8.
 
 Initially, follow the [Quickstart](/install/packstack/) but stop when you see the first "packstack --allinone" at Step 3, instead do:
 
-    # packstack --allinone --provision-demo=n --os-neutron-ovs-bridge-mappings=extnet:br-ex --os-neutron-ovs-bridge-interfaces=br-ex:eth0 --os-neutron-ml2-type-drivers=vxlan,flat
+    # packstack --allinone --provision-demo=n --os-neutron-ovn-bridge-mappings=extnet:br-ex --os-neutron-ovn-bridge-interfaces=br-ex:eth0
+
+Since stein release packstack runs neutron with ovn backend, if want to deploy with ovs as neutron backend, try:
```

Comment:
this sentence is not clear. Should one execute line 14 or line 18?

Reply:
Based on requirement need to do either of L14 or 18. I have modified the sentence